shared-lib/spiutils/src/protocol/firmware.rs

```diff
@@ -371,9 +365,6 @@ impl ToWire for WriteChunkRequest<'_> {
 wire_enum! {
     /// The result of a write chunk request.
     pub enum WriteChunkResult: u8 {
-        /// Unknown result type.
-        Unknown = 0xff,
-
         /// Success
         Success = 0x00,
```

Comment:
Would it make more sense for Success=1 and Failure=2, to more easily catch uninitialized values?  Or is that impossible in rust?

Reply:
See above.